### ISSUE TITLE
delete and deleteMany are not working for models with and without relations

### ISSUE THREAD

**Issue by naveen-bharathi** [prisma/photonjs#382]
#### **`delete`** does not delete relations even when using **`@relation(onDelete: CASCADE)`**

``` prisma
model One {
  id        String   @id @default(cuid())
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
  FOne      DateTime
  FTwo      Two      @relation(name: "OneToTwo", onDelete: CASCADE)
}

model Two {
  id        String   @id @default(cuid())
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
  FOne      String   @unique
  FTwo      String
}
``` 
When deleting Model One, Model Two is not deleted.

**Comment by pantharshit00** [comment]
> delete does not delete relations even when using @relation(onDelete: CASCADE)

This is expected right now as cascade deletes are not implemented. Right now you can manually edit the schema and set the cascade behviuor to work around this.


> Today I ran into another problem. delete and deleteMany are not even working for models without relation

I am unable to reproduce this. I tested the exact same query and it deleted the required records from the database.
<img width="637" alt="image" src="https://user-images.githubusercontent.com/22195362/72278664-808e7e00-365a-11ea-8a6f-c2d6424053ee.png">
<img width="1407" alt="image" src="https://user-images.githubusercontent.com/22195362/72278672-84ba9b80-365a-11ea-8f46-4098845b1728.png">


So if please try second query again making sure you are properly awaiting the Promises as pointed out by @JasonZStanley. If this is resolved we can close this as a duplicate of #702.

**Comment by naveen-bharathi** [comment]
You can close this issue.
